app/views/admin/accounts/edit.html.erb

```diff
@@ -21,7 +21,15 @@
             <%= f.text_field :tenant, class: 'form-control', readonly: @account.persisted? %>
           </div>
           <% is_superadmin = current_ability.superadmin? %>
-          <% current_account.public_settings(is_superadmin: is_superadmin).each do |key, value| %>
+          <% settings = current_account.public_settings(is_superadmin: is_superadmin) %>
+          
+          <!-- Google Analytics Settings Section -->
+          <% if settings.key?(:analytics) || settings.key?(:google_analytics_id) || settings.key?(:google_analytics_property_id) %>
```

Comment:
what happens if its motombo and not google analytics?

Reply:
aw dang, I didn't think of that 😓 🤔